_includes/people.html

```diff
@@ -50,18 +50,18 @@
       </div>
     </div>
 
-    {% if user.roles or user.title %}
+    {% if roles or user.title %}
```

Comment:
```suggestion
    {% if ols-roles or user.title %}
```

Reply:
Not 100% sure it will work

Reply:
I already tried this and it did not work.

Reply:
With the 2 suggestions I just made, it works for me locally